app/enterprise/changelog.md

```diff
@@ -479,6 +495,35 @@ no_version: true
   with command not found". [#7523](https://github.com/Kong/kong/pull/7523)
 
 
+## 2.4.1.2
+**Release Date** 2021/08/12
+
+### Fixes
+
+#### Enterprise
+- Users can now successfully delete workspaces after deleting all entities associated with that workspace.
+  Previously, Kong Gateway was not correctly cleaning up parent-child relationships. For example, creating
+  an Admin also creates a Consumer and RBAC user. When deleting the Admin, the Consumer and RBAC user are
+  also deleted, but accessing the `/workspaces/workspace_name/meta` endpoint would show counts for Consumers
+  and RBAC users, which prevented the workspace from being deleted. Now deleting entities correctly updates
+  the counts, allowing an empty workspace to be deleted.
+- Updates Kong Dev Portal templates' JQuery dependency to v3.6.0, improving security.
+- Users with the `kong_admin` role can now log in to Kong Manager when `enforce_rbac=both` is set. 
+- Renamed the property identifying control planes in hybrid mode when using Kong Vitals with anonymous
+  reports enabled. Before, users received the error, `Cannot use this function in data plane`, on their control planes.
+
+#### Plugins
+- [OpenID Connect](/hub/kong-inc/openid-connect) (`openid-connect`)
+  With this fix, when the OpenID Connect plugin is configured with a `config.anonymous` consumer and
+  `config.scopes_required` is set, a token missing the required scopes will have the anonymous consumer
+  headers set when sent to the upstream.
+- Fixes a regression in the Kong Collector plugin that caused HARs to be kept in its queue indefinitely.
+
+#### Hybrid Mode
+- Control planes are now more lenient when checking data planes' compatibility in hybrid mode. See the
+  [Version compatibility](/gateway-oss/2.5.x/hybrid-mode/#version_compatibility)
```

Comment:
```suggestion
  [Version compatibility](/enterprise/2.4.x/deployment/hybrid-mode/#version-compatibility)
```

Not sure if that is correct or not (matched the 2.5.x, but with 2.4.x); feel free to disregard.

Reply:
ah, yeah you are right that is the correct link address - though I will need to update those docs as the section updates were added to the 2.5.x release docs and not the 2.4.x and 2.3.x - will include update in PR.